src/traffic_layout/engine.cc

```diff
@@ -227,7 +227,7 @@ LayoutEngine::create_runroot()
     empty_check_directory = ts_runroot;
     if (ftw(ts_runroot.c_str(), check_directory_empty, OPEN_MAX_FILE) != 0) {
       // if the directory is not empty, check for valid Y/N
-      for (int i = 0; i < 3; i++) {
+      for (int i = 0; i < 3; i++) { // lgtm[cpp/constant-comparison]
```

Comment:
This can just be:
```
     for (int i = 0; true; i++) {
```
because the if statement at line 240 guarantees that i will never be 3.

Reply:
Right. This is just checking 'Y/N'...